modules/monitoring/application/views/scripts/partials/comment/comment-detail.phtml

```diff
@@ -79,4 +79,6 @@
     } ?>
     </span>
 </div>
-<?= $this->nl2br($this->markdown($comment->comment, isset($textId) ? ['id' => $textId] : null)) ?>
+<span class="caption">
+    <?= $this->nl2br($this->markdown($comment->comment, isset($textId) ? ['id' => $textId] : null)) ?>
+</span>
```

Comment:
You can't use a `span` here, it's an inline element and cannot contain block elements such as `section`. Just pass the `caption` class directly to the helper, it's also on the `section` anyway in Icinga DB Web.